datajunction-clients/python/datajunction/compile.py

```diff
@@ -809,10 +819,7 @@ async def load_data(
                 definition=definition,
                 path=str(path),
             )
-    raise DJClientException(
```

Comment:
I removed this because in exporting/importing yaml files, they were often in directories that already had other, irrelevant and non-compatible yaml files. This is especially prevalent when running from Jupyter, which would create checkpoint yaml files that could be safely ignored.

Reply:
I added this to prevent overwriting local changes to a project by accidentally importing over it.